### PR TITLE
Ronniegeraghty release csv updates2

### PR DESCRIPTION
**Pull request by ronniegeraghty**
Fixes the release CSVs for: 
- Healthcare APIs 
- App Compliance Automation
- Application Insights
- Azure Stack
- Core Packages
- AAD External Identities
- Azure Arc Data
- BareMetal Infrastructure
- JS Track 1 MGMT Packages

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -143,7 +143,7 @@
 "Azure.ResourceManager.ContainerInstance","1.0.0","","Resource Management - Container Instance","Container Instance","containerinstance","","","mgmt","true","","08/29/2022","08/29/2022","","","","","","","",""
 "Azure.ResourceManager.ContainerRegistry","1.0.2","1.1.0-beta.3","Resource Management - Container Registry","Container Registry","containerregistry","","","mgmt","true","","10/21/2022","08/12/2022","","","","","","","",""
 "Azure.ResourceManager.ContainerService","1.0.0","","Resource Management - Container Service","Container Service","containerservice","","","mgmt","true","","09/26/2022","09/26/2022","","","","","","","",""
-"Azure.ResourceManager","1.3.2","","Resource Management - Core","Azure Resource Manager","resourcemanager","","","mgmt","true","","11/11/2022","04/07/2022","","","","","","","",""
+"Azure.ResourceManager","1.3.2","","Resource Management - Core","Azure Resource Manager","resourcemanager","","","core","true","","11/11/2022","04/07/2022","","","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:146]: Should this be Other like Azure.Core? Also what is intended with the new core type?

**Review by weshaggard — APPROVED** [review]
One question both otherwise looks good.

File: _data/releases/latest/js-packages.csv [hunk 2]
```diff
@@ -541,79 +541,78 @@
...
-"azure-asm-storage","","","","Storage","","NA","NA","mgmt","","","","","","","","","","","","Deprecated. Added for MSDocs ToC generation. Not published by azure-sdk automation."
-"azure-asm-trafficmanager","","","","Traffic Manager","","NA","NA","mgmt","","","","","","","","","","","","Deprecated. Added for MSDocs ToC generation. Not published by azure-sdk automation."
-"azure-asm-website","","","","App Service","","NA","NA","mgmt","","","","","","","","","","","","Deprecated. Added for MSDocs ToC generation. Not published by azure-sdk automation."
-"@azure/arm-changeanalysis","1.1.1","","@azure/arm-changeanalysis","Changeanalysis","changeanalysis","","NA","mgmt","true","","","","deprecated","3/31/2023","true","@azure/arm-changeanalysis","https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md","","",""
+"azure-arm-advisor","1.2.0","","Resource Management - Advisor","Advisor","","NA","NA","mgmt","false","","","","deprecated","","","@azure/arm-advisor","","","","Deprecated. Added for MSDocs ToC generation. Not published by azure-sdk automation."
```
Comment (weshaggard) [inline _data/releases/latest/js-packages.csv:544]:
```suggestion
"azure-arm-advisor","1.2.0","","Resource Management - Advisor","Advisor","NA","NA","NA","mgmt","false","","","","deprecated","","","@azure/arm-advisor","","","","Deprecated. Added for MSDocs ToC generation. Not published by azure-sdk automation."
```

You probably need to add NA for the source link for most of these to fix those breaks.

Comment (weshaggard) [reply _data/releases/latest/js-packages.csv:544]: At least for anywhere that has and empty RepoPath set it to NA.

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (ronniegeraghty) [reply _data/releases/latest/dotnet-packages.csv:146]: Didn't know `Other` was the type for core packages. Will change.

Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv]: To be clear Other is the service name for Core. The type is still client.

File: _data/releases/latest/dotnet-packages.csv [hunk 3]
```diff
@@ -143,7 +143,7 @@
 "Azure.ResourceManager.ContainerInstance","1.0.0","","Resource Management - Container Instance","Container Instance","containerinstance","","","mgmt","true","","08/29/2022","08/29/2022","","","","","","","",""
 "Azure.ResourceManager.ContainerRegistry","1.0.2","1.1.0-beta.3","Resource Management - Container Registry","Container Registry","containerregistry","","","mgmt","true","","10/21/2022","08/12/2022","","","","","","","",""
 "Azure.ResourceManager.ContainerService","1.0.0","","Resource Management - Container Service","Container Service","containerservice","","","mgmt","true","","09/26/2022","09/26/2022","","","","","","","",""
-"Azure.ResourceManager","1.3.2","","Resource Management - Core","Azure Resource Manager","resourcemanager","","","mgmt","true","","11/11/2022","04/07/2022","","","","","","","",""
+"Azure.ResourceManager","1.3.2","","Resource Management - Core","Azure Resource Manager","resourcemanager","","","other","true","","11/11/2022","04/07/2022","","","","","","","",""
```
[the remark this answers is not in the document]
Comment (ronniegeraghty) [reply _data/releases/latest/dotnet-packages.csv]: Kept the type `mgmt` and switched Service to `Core`. I'll try to put any internal packages in the `Core` Service

[the remark this answers is not in the document]
Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv]: Azure.Core is still in the Other service group not Core. We should probably make this one consistent.

[the remark this answers is not in the document]
Comment (ronniegeraghty) [reply _data/releases/latest/dotnet-packages.csv]: Will add these in my next PR

**Pull request merged**